src/helpers/error-message.helper.ts

```diff
@@ -29,52 +30,61 @@ export class ErrorMessageHelper {
     let message: string;
     let title = "Nie rozpoznano błędu";
     let day = 0;
+    let type = 1;
 
     switch (error.kind) {
       case AlgorithmErrorCode.AON:
         message = `Brak pielęgniarek w dniu ${error.day} na zmianie ${
           error.day_time ? dayTimeTranslations[error.day_time] : ""
         }`;
+        type = ScheduleErrorType.NOT_ENOUGH_WORKERS;
```

Comment:
Może lepiej by zrobić to w taki sposób, że typy błędów to będą typy z algorytmu i tyle będziemy miec sekcji. Co o tym myślisz @prenc?

Reply:
Można to jednolicić, w readme solvera można znaleźć jak to tam wygląda. Nie wiem o co chodzi z _tyle będziemy miec sekcji_.